integrations/tests/test_shuffle.py

```diff
@@ -0,0 +1,95 @@
+# Copyright (C) 2015, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute
+# it and/or modify it under the terms of GPLv2
+
+"""Unit tests for shuffle.py integration."""
+
+import sys
+import os
+import pytest
+import shuffle
+import random
+
+sys.path.append(os.path.join(os.path.dirname(
+    os.path.realpath(__file__)), '..', '..'))
+
+filter_values = ["87924", "87900", "87901", "87902", "87903", "87904", "86001",
+                 "86002", "86003", "87932", "80710", "87929", "87928",
+                 "5710"]
+
+
+@pytest.mark.parametrize('json_alert, msg', [({'timestamp': 'year-month-dayThours:minuts:seconds+0000',
+                                               'rule': {'level': 0, 'description': 'alert description',
+                                                        'id': 'rule-id',
+                                                        'firedtimes': 1},
+                                               'id': 'alert_id',
+                                               'full_log': 'full log.', 'decoder': {'name': 'decoder-name'},
+                                               'location': 'wazuh-X'},
+                                              '{"severity": 1, "pretext": "WAZUH Alert", "title": "alert description", "text": "full log.", "rule_id": "rule-id", "timestamp": "year-month-dayThours:minuts:seconds+0000", "id": "alert_id", "all_fields": {"timestamp": "year-month-dayThours:minuts:seconds+0000", "rule": {"level": 0, "description": "alert description", "id": "rule-id", "firedtimes": 1}, "id": "alert_id", "full_log": "full log.", "decoder": {"name": "decoder-name"}, "location": "wazuh-X"}}'),
+                                             ({'timestamp': 'year-month-dayThours:minuts:seconds+0000',
+                                               'rule': {'level': 0, 'description': 'alert description',
+                                                        'id': random.choice(filter_values),
+                                                        'firedtimes': 1},
+                                               'id': 'alert_id',
+                                               'full_log': 'full log.', 'decoder': {'name': 'decoder-name'},
+                                               'location': 'wazuh-X'},
+                                              '')])
+def test_generate_msg(json_alert, msg):
```

Comment:
Do not hardcode events in the parametrize. The test cases you are using are identical except for the `rule-id`. Use variables for both the alert and the expected message and provide only the rule_id.

Take this as an example on how it could look like:

```suggestion
@pytest.mark.parametrize('alert, expected_msg, rule_id', [
    (alert_template, msg_template, 1),
    (alert_template, "", shuffle.SKIP_RULE_IDS[0])
])
def test_generate_msg(alert, expected_msg, rule_id):
```